docs/contribute-how-to.md

```diff
@@ -5,10 +5,242 @@ mentions:
 -   TheItsNameless
 ---
 
-# *Under Construction*
+# About this Guide
 
-Here you can learn how to contribute to our wiki!
+This guide will walk you trough the downloading and setup of the software you need to work on our wiki, the steps to clone this wiki, so you can edit it locally and how to create a `Pull Request` to add your changes to the wiki.
 
-Well, actually this page doesn't exist at the moment, but look back later! We are working on it the moment you read this!
+This article will teach you how to use GitHub Desktop, the easiest way to manage your local Repositories. Its possible to do this without GitHub Desktop just by using Git, but this is more advanced than just using GitHub Desktop and we won't be covering this in this guide.
 
-If you have a question right now, join our [Discord Server](/discord) and ask it! We will help you.
+:::tip
+If you already have installed **VSCode**, **GitHub Desktop** and **NodeJS** and want to learn more about forking and cloning a GitHub Repository, you can continue 
+
+<BButton link='/contribute-how-to#setup-your-local-environment' color=blue>Setup local environment</BButton>
+:::
+
+:::tip
+If you already have installed the necessary software and know how to fork and clone a repository, but want to learn more about committing and Pull Requests, you can continue
+
+<BButton link='/contribute-how-to#committing-your-changes' color=blue>Committing and Pull Requests</BButton>
+:::
+
+:::tip
+If you already have installed the necessary software and know how to handle Git-Repositories and GitHub, you can continue by learning how to style our pages!
+
+<BButton link='/contribute-style' color=blue>Style our pages</BButton>
+:::
+
+## Steps to edit the wiki
+
+To edit the wiki, there are 5 major steps that you need to follow (downloading and setting up the software is just done the first time):
+
+:::tip
+The 5 steps that you need to follow to successfully update our wiki
+
+1. Update your fork to match the latest Version of our Wiki
+2. Fetch the latest changes in GitHub Desktop
+3. Edit the Wiki
+4. Commit your local changes and sync with GitHub
+5. Create a Pull Request
+:::
+
+If you see this the first time, it might look a little overwhelming, but don't worry! Its very easy and we guide you through this step by step.
+
+# Setting up the Software
```

Comment:
Do you mean that i should add one level to each heading? Make Headings in level 1 to level 2, in level 2 to level 3, etc.

Reply:
Its kind of up to you. Just be aware how it effects the styling. 

MOST articles do not use h1 at all, since thats reserved for the top level title of the page. So normally the max to use is h2, and then h3. 

A very few select articles use h1, such as "3D items" page, since there are actually two distinct "mini articles" inside one main article.